<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1062</width>
    <height>748</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MediaElch</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralWidget {
	background-image: url(:/img/pattern2.png);
}

#menuWidget {
	background-color: rgb(51, 57, 61);
	border-right: 1px solid rgb(34, 34, 34);
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="menuWidget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>28</number>
       </property>
       <property name="leftMargin">
        <number>16</number>
       </property>
       <property name="topMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>16</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="buttonMovies">
         <property name="toolTip">
          <string>Movies</string>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 0;</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="MediaElch.qrc">
           <normaloff>:/img/video_menuActive.png</normaloff>:/img/video_menuActive.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="buttonMovieSets">
         <property name="toolTip">
          <string>Movie Sets</string>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 0;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="MediaElch.qrc">
           <normaloff>:/img/movieSets_menu.png</normaloff>:/img/movieSets_menu.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="buttonTvshows">
         <property name="toolTip">
          <string>TV Shows</string>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 0;</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="MediaElch.qrc">
           <normaloff>:/img/display_on_menu.png</normaloff>:/img/display_on_menu.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="buttonSettings">
         <property name="toolTip">
          <string>Settings</string>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 0;</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="MediaElch.qrc">
           <normaloff>:/img/spanner_menu.png</normaloff>:/img/spanner_menu.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>28</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSplitter" name="movieSplitter">
          <property name="styleSheet">
           <string notr="true">QSplitter::handle {
	background-color: transparent;
}
</string>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="handleWidth">
           <number>1</number>
          </property>
          <property name="childrenCollapsible">
           <bool>false</bool>
          </property>
          <widget class="FilesWidget" name="filesWidget" native="true">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
          </widget>
          <widget class="MovieWidget" name="movieWidget" native="true"/>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSplitter" name="tvShowSplitter">
          <property name="styleSheet">
           <string notr="true">QSplitter::handle {
	background-color: transparent;
}
</string>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="handleWidth">
           <number>1</number>
          </property>
          <property name="childrenCollapsible">
           <bool>false</bool>
          </property>
          <widget class="TvShowFilesWidget" name="tvShowFilesWidget" native="true">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
          </widget>
          <widget class="TvShowWidget" name="tvShowWidget" native="true"/>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="Settings" name="settings"/>
      <widget class="SetsWidget" name="setsWidget"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MovieWidget</class>
   <extends>QWidget</extends>
   <header>movies/MovieWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FilesWidget</class>
   <extends>QWidget</extends>
   <header>movies/FilesWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TvShowFilesWidget</class>
   <extends>QWidget</extends>
   <header>tvShows/TvShowFilesWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TvShowWidget</class>
   <extends>QWidget</extends>
   <header>tvShows/TvShowWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SetsWidget</class>
   <extends>QWidget</extends>
   <header>sets/SetsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Settings</class>
   <extends>QWidget</extends>
   <header>settings/Settings.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="MediaElch.qrc"/>
 </resources>
 <connections/>
</ui>
